actioncable/test/remote_connections_test.rb

```diff
@@ -0,0 +1,20 @@
+require 'test_helper'
+require 'stubs/test_server'
+
+class ActionCable::RemoteConnections::BaseTest < ActionCable::TestCase
+  setup do
+    @server             = TestServer.new
+    @remote_connections = ActionCable::RemoteConnections.new(@server)
+  end
+
+  test '::new' do
+    assert_instance_of ActionCable::RemoteConnections, @remote_connections
+    assert @remote_connections.respond_to?(:server)
+    assert_equal @server, @remote_connections.server
+  end
+
+  test '#where' do
+    remote_connection = @remote_connections.where(var1: 1, var2: 2, var3: 3)
+    assert_instance_of ActionCable::RemoteConnections::RemoteConnection, remote_connection
```

Comment:
Here we can assert that values of `var`, `var2` and `var3` are returned as `connection_identifier`